pyhf/modifiers/normsys.py

```diff
@@ -31,7 +31,7 @@ def expected_data(self, pars):
 
     def pdf(self, a, alpha):
         tensorlib, _ = get_backend()
-        return tensorlib.normal(a, alpha, tensorlib.astensor([1]))
+        return getattr(tensorlib, self.pdf_type)(a, alpha, tensorlib.astensor([1]))
```

Comment:
here we see that the signature of the pdf_type obviously depends on the pdf used, we can just trust that this is correct and introduce checks later?

Reply:
Good point, I can always grab the current tensorlib and validate at the time of modifier definition.

Reply:
how woud you validate it? seems like you'd need to analyze the implementation of `def pdf` to check if the signature called matches the signature that's defined